src/main/java/com/intuit/graphql/orchestrator/batch/EntityFetcherBatchResultTransformer.java

```diff
@@ -31,31 +33,13 @@ public EntityFetcherBatchResultTransformer(String providerNamespace, String enti
     @Override
     public List<DataFetcherResult<Object>> toBatchResult(DataFetcherResult<Map<String, Object>> dataFetcherResult,
                                                          List<DataFetchingEnvironment> dataFetchingEnvironments) {
-        if(dataFetcherResult.hasErrors()) {
-            throw EntityFetchingException.builder()
-                    .serviceNameSpace(providerNamespace)
-                    .fieldName(extFieldName)
-                    .parentTypeName(entityName)
-                    .additionalInfo(
-                        dataFetcherResult.getErrors().stream()
-                        .map(GraphQLError::getMessage)
-                        .reduce("", (partialString, errMsg) -> StringUtils.join(partialString, errMsg, " "))
-                    )
-                    .build();
-        }
-
         List<DataFetcherResult<Object>> dataFetcherResults = new ArrayList<>();
         List<Map<String, Object>> _entities = (MapUtils.isEmpty(dataFetcherResult.getData())) ? Collections.emptyList() : (List<Map<String, Object>>) dataFetcherResult.getData().get(_ENTITIES_FIELD_NAME);
-        if(CollectionUtils.isNotEmpty(_entities)) {
-            _entities.forEach(entityResult -> {
-                Object fieldData = (entityResult != null) ? entityResult.get(this.extFieldName) : null;
-
-                dataFetcherResults.add(DataFetcherResult.newResult()
-                        .data(fieldData)
-                        .build()
-                );
-            });
-        } else {
+
+        if(CollectionUtils.isEmpty(_entities)) {
+            //Should not happen; if it did, downstream did not create entity fetcher correctly or is not using DGS
+            log.warn("Provider {} is not using DGS or invalid version", providerNamespace);
```

Comment:
I suggest log: "Provider {} is not Federation subgraph specification compliant.  _entities field missing or empty in the response."